mungegithub/mungers/sync/issue-sync.go

```diff
@@ -105,25 +131,26 @@ func (s *IssueSyncer) Sync(source IssueSource) error {
 		return nil
 	}
 
+	var obj *github.MungeObject
 	// Update an issue if possible.
 	if len(updatableIssues) > 0 {
-		obj := updatableIssues[0]
+		obj = updatableIssues[0]
 		// Update the chosen issue
-		if err := s.updateIssue(obj, source); err != nil {
+		if err = s.updateIssue(obj, source); err != nil {
 			return fmt.Errorf("error updating issue %v for %v: %v", *obj.Issue.Number, source.ID(), err)
 		}
-		s.synced.Insert(source.ID())
-		return nil
-	}
-
-	// No issue could be updated, create a new issue.
-	n, err := s.createIssue(source)
-	if err != nil {
-		return fmt.Errorf("error making issue for %v: %v", source.ID, err)
+	} else {
+		// No issue could be updated, create a new issue.
+		obj, err = s.createIssue(source)
+		if err != nil {
+			return fmt.Errorf("error making issue for %v: %v", source.ID, err)
+		}
+		issueNum := *obj.Issue.Number
+		s.finder.Created(source.Title(), issueNum)
 	}
-	s.finder.Created(source.Title(), n)
 	s.synced.Insert(source.ID())
-	return nil
+
+	return s.autoPrioritize(obj)
```

Comment:
nit: I slightly prefer the old logic. You know when you first file an issue that there's only one occurrence, so you could just file it with the P3 label or whatever-- no need to auto prioritize?